icons/home.svg

```diff
@@ -9,6 +9,6 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="m3 9 9-7 9 7v11a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2z" />
-  <polyline points="9 22 9 12 15 12 15 22" />
+  <path d="M3 19A2 2 0 0 0 5 21h14a2 2 0 0 0 2-2v-9a2 2 0 0 0-.772-1.579L12 2 3.772 8.421A2 2 0 0 0 3 10z" />
+  <rect x="9" y="12" width="6" height="9" rx="1" />
```

Comment:
Shouldn't the top corner also be rounded?

```suggestion
  <path d="M15 21v-8a1 1 0 0 0-1-1h-4a1 1 0 0 0-1 1v8" />
  <path d="M3 9.923a2 2 0 0 1 .709-1.451l7-5.999a2 2 0 0 1 2.582 0l7 5.999A2 2 0 0 1 21 10.001V19a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2.001z" />
```

<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M15+21v-8a1+1+0+0+0-1-1h-4a1+1+0+0+0-1+1v8%22+/%3E%3Cpath+d=%22M3+9.923a2+2+0+0+1+.709-1.451l7-5.999a2+2+0+0+1+2.582+0l7+5.999A2+2+0+0+1+21+10.001V19a2+2+0+0+1-2+2H5a2+2+0+0+1-2-2.001z%22+/%3E&name=image"><img alt="icons" width="400" src="https://lucide.dev/api/gh-icon/PHN2ZwogIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIKICB3aWR0aD0iMjQiCiAgaGVpZ2h0PSIyNCIKICB2aWV3Qm94PSIwIDAgMjQgMjQiCiAgZmlsbD0ibm9uZSIKICBzdHJva2U9ImN1cnJlbnRDb2xvciIKICBzdHJva2Utd2lkdGg9IjIiCiAgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIgogIHN0cm9rZS1saW5lam9pbj0icm91bmQiCj4KICA8cGF0aCBkPSJNMTUgMjF2LThhMSAxIDAgMCAwLTEtMWgtNGExIDEgMCAwIDAtMSAxdjgiIC8+CiAgPHBhdGggZD0iTTMgOS45MjNhMiAyIDAgMCAxIC43MDktMS40NTFsNy01Ljk5OWEyIDIgMCAwIDEgMi41ODIgMGw3IDUuOTk5QTIgMiAwIDAgMSAyMSAxMC4wMDFWMTlhMiAyIDAgMCAxLTIgMkg1YTIgMiAwIDAgMS0yLTIuMDAxeiIgLz4KPC9zdmc+.svg"/><br/>Open lucide studio</a>

![image](https://github.com/lucide-icons/lucide/assets/17746067/5f3282be-7ce9-48d9-9389-85524942a680)